packages/web3-constants/evm/coinmarketcap.json

```diff
@@ -13,6 +13,7 @@
         "Arbitrum_Rinkeby": "",
         "xDai": "5601",
         "Celo": "5567",
-        "Fantom": "3513"
+        "Fantom": "3513",
+        "Avalanche": "43114"
```

Comment:
It isn't a chain id. Learn more: https://github.com/DimensionDev/Maskbook/issues/4874